src/cable/server.cr

```diff
@@ -155,7 +182,11 @@ module Cable
       spawn(name: "Cable::Server - process_subscribed_messages") do
         while received = fiber_channel.receive
           channel, message = received
-          server.send_to_channels(channel, message)
+          if channel.includes?("cable_internal") && message == Cable.message(:disconnect)
```

Comment:
I originally tried `starts_with?` but it would never hit this. Then I logged out the channel, and it was `channel:cable_internal/1234-1234`. I'm just not sure where that `channel:` part was coming from.